scripts/Jenkinsfile_batch_airgap_test

```diff
@@ -0,0 +1,88 @@
+#!groovy
+
+node {
+  def branch = "main"
+  def test_dir = "${env.TEST_DIRECTORY_LIST}".split(',')
+
+  if ("${env.BRANCH}" != "null" && "${env.BRANCH}" != "") {
+    branch = "${env.BRANCH}"
+  }
+
+  def repo = scm.userRemoteConfigs
+  if ("${env.REPO}" != "null" && "${env.REPO}" != "") {
+    repo = [
+      [url: "${env.REPO}"]
+    ]
+  }
+  def timeout = "60m"
+  if ("${env.TIMEOUT}" != "null" && "${env.TIMEOUT}" != "") {
+    timeout = "${env.TIMEOUT}"
+  }
+  wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm', 'defaultFg': 2, 'defaultBg': 1]) {
+    withFolderProperties {
+      paramsMap = []
+      params.each {
+        if (it.value && it.value.trim() != "") {
+          paramsMap << "$it.key=$it.value"
+        }
+      }
+      withEnv(paramsMap) {
+        stage('Checkout') {
+          deleteDir()
+          checkout([
+            $class: 'GitSCM',
+            branches: [
+              [name: "*/${branch}"]
+            ],
+            extensions: scm.extensions + [
+              [$class: 'CleanCheckout']
+            ],
+            userRemoteConfigs: repo
+          ])
+        }
+        dir("./") {
+          stage('Run Parallel Jobs') {
+            echo "Running ${test_dir.size()} tests: ${test_dir}"
+            def jobs = [:]
+
+            test_dir.each { test_name ->              
+              def actual_testdir_name = "airgap"
+              def job_test_args = "${TEST_ARGS}"
+              def job_aws_ssh_key_name = "${AWS_SSH_KEY_NAME}"
+              def job_aws_ssh_pem_key = "${AWS_SSH_PEM_KEY}"
+
+              if (test_name == "private_registry") {
+                job_test_args = job_test_args + " -tags=privateregistry"
+              } else if ((test_name == "system_default_registry")) {
+                job_test_args = job_test_args + " -tags=systemdefaultregistry"
+              } else if ((test_name == "tarball")) {
+                job_test_args = job_test_args + " -tags=tarball -tarballType tar.zst"
```

Comment:
just a curious thought, is it possible to add a job for tarballType tar.gz as well? 
like from the jenkinsjob - we can give only one testname. 
here we can split it into 2 tests - one for tar.gz and one for tar.zst?

Reply:
We don't need 2 tests as we have .gz because some old versions didn’t support .zst. There is no change in code.